src/components/views/game/game/GameView.scss

```diff
@@ -33,8 +33,8 @@
     position: absolute;
     left: 3%;
     bottom: 3%;
-    height: 50%;
-    width: 50%;
+    height: 340px;
```

Comment:
with an absolute pixel size it may not look good anymore on high resolution screens.
You did not find an appropriate percentage height?

Reply:
I can try, of course!